### PR TITLE
Add taskwarrior package

### PR DESCRIPTION
**Pull request by uberhacker**
Taskwarrior is Free and Open Source Software that manages your TODO list from the command line.
It is flexible, fast, and unobtrusive. It does its job then gets out of your way.  See https://taskwarrior.org/.

Tested & Working properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/uberhacker/chromebrew.git CREW_BRANCH=add-taskwarrior-package crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: packages/taskwarrior.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  depends_on 'glibc' # R
+  depends_on 'util_linux' # R
+
+  def self.postinstall
+    ExitMessage.add "\nType 'man task' for help to get started.\n"
+  end
+
+  def self.postremove
+    taskrc = "#{HOME}/.taskrc"
+    if File.exist? taskrc
```
Comment (satmandu) [inline packages/taskwarrior.rb:31]: We have a postinstall remove macro now!

Comment (uberhacker) [reply packages/taskwarrior.rb:31]: Does it work for files and directories?

Comment (satmandu) [reply packages/taskwarrior.rb:31]: I believe so? Do you want to check?

Comment (uberhacker) [reply packages/taskwarrior.rb:31]: I examined remove.rb and didn't notice any special logic to handle the cleanup.  Also, many packages still have logic similar to this.

Comment (satmandu) [reply packages/taskwarrior.rb:31]: See this in `package.rb`:
```ruby
  def self.agree_to_remove(config_object = nil)
    if File.file? config_object
      identifier = 'file'
    elsif File.directory? config_object
      identifier = 'directory'
    else
      puts "Cannot identify #{config_object}.".lightred
      return
    end
    if agree_default_no("Would you like to remove the config #{identifier}: #{config_object}")
      FileUtils.rm_rf config_object
      puts "#{config_object} removed.".lightgreen
    else
      puts "#{config_object} saved.".lightgreen
    end
  end
```

In `ruby_pry.rb`:
```ruby
  def self.postremove
    Package.agree_to_remove("#{HOME}/.pryrc")
  end
```

In `stellarium.rb`:
```ruby
  def self.postremove
    Package.agree_to_remove("#{HOME}/.stellarium")
  end
```

Is that not cleaner?

Comment (uberhacker) [reply packages/taskwarrior.rb:31]: Gotcha, didn't notice that.

**Review by satmandu — APPROVED** [review]

**Pull request merged**